low-coverage/script.whizzml

```diff
@@ -2,17 +2,13 @@
   (let (ds (fetch dsid)
         size (ds "rows")
         name (ds "name")
-        fields (keys (ds "fields"))
-        missing (map (lambda (x) (ds ["fields" 
-                                          x 
-                                          "summary" 
-                                          "missing_count"])) 
-                         fields)
-        missing-pairs (iterate (a [] x fields y missing) (append a [x y]))
-
-        low-list (filter (lambda (x) (< (last x) (* (- 1 threshold) size))) 
-                         missing-pairs)
-        fields-to-keep (map (lambda (x) (head x)) low-list))
+        fields (ds "fields")
+        mc (if (< threshold 1)
```

Comment:
looks good, but i'd add here also a check to ensure threshold is non-negative, `raise`-ing an error (just a string) otherwise.